front/src/BasicTable.tsx

```diff
@@ -9,42 +9,56 @@ import { useTranslation } from 'react-i18next';
 import { Button } from '@material-ui/core';
 import "./BasicTable.css";
 
-export default function BasicTable() {
-  const [offers, setOffers] = useState<any[]>([]); // TODO fix any
+
+interface OffersTypes {
+      _id: string,
```

Comment:
It's fine, but if you want to be a bit more strict, you can use different type for `_id`. See https://stackoverflow.com/questions/54743461/how-to-define-mongoose-id-in-typescript-interface